test/integration/isolated/suite_test.go

```diff
@@ -325,6 +325,7 @@ func featureSetup(opts ...featureSetupOpt) func(ctx context.Context, t *testing.
 			fmt.Sprintf("--admission-webhook-key=%s", key),
 			fmt.Sprintf("--admission-webhook-listen=0.0.0.0:%d", testutils.AdmissionWebhookListenPort),
 			"--anonymous-reports=false",
+			"--log-level=trace",
```

Comment:
Yes, I used the same config with CLI flags with the original integration tests.
But there may not be difference between `trace` and `debug` because we does not use `TraceLevel` in logs in our code base.